### ISSUE TITLE
Running `localhost run ... ` using the npm command doesn't work on Windows

### ISSUE THREAD

**Issue by glennsarti** [amykapernick/localhost-proxy#8]
On a Windows 10 computer, given a site configuration that has a script like `npm run server`.  Using `localhost run mysite` throws an error like

```
PS> localhost run mysite
Error: spawn npm ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:469:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn npm',
  path: 'npm',
  spawnargs: [ 'run', 'index' ]
}
```

But running npm manually works

```
PS> npm run index

> testproject@1.0.0 index C:\Source\localhost-proxy\tests\fixtures\project
> node ./index.js

DEBUG TEST MESSAGE
ERROR TEST MESSAGE
WARN TEST MESSAGE
INFO TEST MESSAGE
LOG TEST MESSAGE
```

**Comment by amykapernick** [comment]
Yea look I have no idea about building CLI tools, so I was just building in WSL for now. Have you solved this is the PRs?

**Comment by amykapernick** [comment]
Has been fixed in #9

**Comment by glennsarti** [comment]
Yes!

https://github.com/amykapernick/localhost-proxy/pull/9/commits/1b1b1526a454164e50d76b393e8bb4a83621eb67

And tested on Windows.
